src/storage/src/hummock/local_version/pinned_version.rs

```diff
@@ -126,15 +134,31 @@ impl PinnedVersion {
         self.version.id != INVALID_VERSION_ID
     }
 
-    pub fn levels(&self, new_compaction_group_id: CompactionGroupId) -> Vec<&Level> {
+    pub fn levels_by_compaction_groups_id(
+        &self,
+        compaction_group_id: CompactionGroupId,
+    ) -> Vec<&Level> {
         let mut ret = vec![];
-        if let Some(new_levels) = self.version.levels.get(&new_compaction_group_id) {
-            ret.extend(new_levels.l0.as_ref().unwrap().sub_levels.iter().rev());
-            ret.extend(new_levels.levels.iter());
+        if let Some(levels) = self.version.levels.get(&compaction_group_id) {
+            ret.extend(levels.l0.as_ref().unwrap().sub_levels.iter().rev());
+            ret.extend(levels.levels.iter());
         }
         ret
     }
 
+    pub fn levels(&self, table_id: TableId) -> Vec<&Level> {
+        if table_id.table_id() == 0 {
```

Comment:
Should this table_id == 0 wildcard only available in test?

Reply:
> Should this table_id == 0 wildcard only available in test?

not in same crate :-(

Reply:
Did you try `#[cfg(any(test, feature = "test"))]`?

Reply:
> #[cfg(any(test, feature = "test"))]

it can :-)